sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
Aha, you're right. That was the original intention, yes - But even that diff is just an approximation. 

```sql
UPDATE payment_instructions SET due = floor(9.41/(amount)) * amount WHERE due > 9.41;
```

^ That assumes that the user didn't change the tip amount. For example - let's say the user had a 5 dollar tip amount. 

### Payday 1 

Charged: 0
Due: $5

### Payday 2

Attempted to charge, failed
Due: $10 (our old code does this)

Expected behaviour after our script runs is that the due is reset to **$5**. 

What happens if the user changes the tip amount after Payday 2 though? If they changed the tip amount to $1, let's say - we'll end up setting the due to `floor(9.41/(amount))`, i.e. **$9**